include/ISceneNode.h

```diff
@@ -230,11 +230,12 @@ namespace scene
 		directly modify the material of a scene node.
 		\param num Zero based index. The maximal value is getMaterialCount() - 1.
 		\return The material at that index. */
+#ifndef NEW_SHADERS
 		virtual video::SGPUMaterial& getMaterial(uint32_t num)

```

Comment:
I think I nuked this function in `master`

Reply:
@Crisspl didn't I?

Reply:
maybe i messed up the merge a little, doesnt really matter since its inside #ifndef

Reply:
this makes me ask how much you could have messed up the merge.